FiveCalls/FiveCalls/Issue.swift

```diff
@@ -28,7 +28,7 @@ struct Issue: Identifiable, Decodable {
     }
     
     var shareURL: URL {
-        return URL(string: String(format: "https://5calls.org/issue/%s/",self.slug))!
+        return URL(string: String(format: "https://5calls.org/issue/%@/",self.slug))!
```

Comment:
@nickoneill did the slugs recently change? I swear this worked with `%s` when I first tested, but then subsequently I was getting weirdly formatted share URL's soon after, but `%@` fixes it.

Reply:
I moved this around in the last ~month, I probably formatted it like I was writing Go unintentionally